packages/blocks/src/api/raw-handling/ms-strip-footnotes.js

```diff
@@ -0,0 +1,22 @@
+export default function( node ) {
+	// Only filter on DOM elements
+	if ( ! node.getAttribute ) {
+		return;
+	}
+
+	// Filters are all based on 'style' attributes
+	const style = node.getAttribute( 'style' );
+
+	// Remove "<a style='mso-footnote-id...'>...</a>"
+	// Note: for now we are just stripping out footnotes
+	if ( style && style.indexOf( 'mso-footnote-id' ) !== -1 ) {
```

Comment:
Do these appear by any chance in the `node.style` object?

Reply:
Hm, I guess that doesn't work. In that case, it would be good to split the value instead on `;` and check the presence of these keys. I don't like the idea of stripping out content so much. Can't we just leave it?